pmo/order/new/new.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @module {Module} new <pmo-order-new>
```

Comment:
I believe the type here should be `{Constructor}`. Even though it's not used that way, that's what it really is.